Changelog.md

```diff
@@ -8,6 +8,8 @@ Changelog
 * Add a timeout for the reset retry request.
 * Add Esperanto translations.
 * Fixed incorrect confirmation url.
+* [BC break] Use UserManager::getRepository() instead of UserManager::repository
+* [BC break] Use UserManager::getClass() instead of UserManager::class
```

Comment:
missing ``$`` for the property name.

And please add backticks around code spans